crates/dbs-virtio-devices/src/mmio/mmio_v2.rs

```diff
@@ -339,8 +339,8 @@ where
 impl<AS, Q, R> DeviceIo for MmioV2Device<AS, Q, R>
 where
     AS: 'static + GuestAddressSpace + Clone + Send + Sync,
-    Q: QueueStateT + Send + Clone,
-    R: GuestMemoryRegion + Send + Sync,
+    Q: QueueStateT + Send + Clone + 'static,
+    R: GuestMemoryRegion + Send + Sync + 'static,
```

Comment:
`'static + QueueStateT + Send + Clone` to keep same style of above line?

Reply:
fixed